### DISCUSSION TITLE
Testing on Windows (without a Windows machine)

### DISCUSSION THREAD

**Discussion by sethpeasley** [tui-cs/Terminal.Gui#1288] [Q&A] [answered]
Hi! 

I like this project and I'd like to contribute, possibly by adding some unit tests. Your guidelines say, "Test on both Windows and Linux", which I want to abide by, but I run Ubuntu and I don't have access to a Windows 10 machine. 

Is there a way to run tests for Windows in some other way?

**Comment by tig** [comment, marked as answer]
Don’t sweat it. Just note in your PRs that you were unable to test on Windows and I’ll give ‘er a whirl before merging.

From: Seth Peasley ***@***.***>
Sent: Thursday, May 6, 2021 11:50 AM
To: migueldeicaza/gui.cs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [migueldeicaza/gui.cs] Testing on Windows (without a Windows machine) (#1288)


Hi!

I like this project and I'd like to contribute, possibly by adding some unit tests. Your guidelines say, "Test on both Windows and Linux", which I want to abide by, but I run Ubuntu and I don't have access to a Windows 10 machine.

Is there a way to run tests for Windows in some other way?

**Reply by sethpeasley** [reply]
I will. Thanks!
